cloudtrail-eventbridge-lambda-dynamodb-sam/template.yaml

```diff
@@ -0,0 +1,63 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: Serverless Webform Sample
+
+Parameters:
+  NotificationEmail:
+    Type: String
+    Description: "Email where notifications should be sent when route advertisements have been blocked."
+
+Resources:
+  SubmitFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: src/function.py
+      Description: "Removes public route advertisements for networks not permitted to be advertised."
+      Handler: function.lambda_handler
+      PackageType: "Zip"
+      Tracing: Active
+      Runtime: "python3.9"
```

Comment:
Please use the latest available runtime version for Python on Lambda which is 3.12